src/json_output.rs

```diff
@@ -209,7 +227,14 @@ impl<W: Write> BinXmlOutput<W> for JsonOutput<W> {
 
     fn visit_characters(&mut self, value: &BinXmlValue) -> Result<()> {
         trace!("visit_chars {:?}", &self.stack);
-        let current_value = self.get_or_create_current_path();
+        // We use the key name if we are separating Element attributes
```

Comment:
I think this flow might not be necessarily needed.

If `seperate_attributes` is enabled, the attributes should have been inserted into a sibling object (of `Element`), and the code in the `mem::replace` branch should run.

Reply:
I think this is still needed:

With logic:
`"EventID":902,"EventID_attributes":{"Qualifiers":16384}`

Without logic:
`"EventID":{"EventID":902},"EventID_attributes":{"Qualifiers":16384}`

Reply:
https://github.com/forensicmatt/evtx/commit/c3b9d8e26693c5da061a4430f000b8c73bf5f943